charts/ui-plugin-server/values.yaml

```diff
@@ -1,47 +1,34 @@
 # Default values for sample.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
-
 replicaCount: 1
-
 image:
-  repository: yourrepohere/plugin-server
+  repository: rancher/homepage
   pullPolicy: Always
   # Overrides the image tag whose default is the chart appVersion.
-  tag: "v0"
-
+  tag: "0.1.0"
 imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
-
 service:
   type: ClusterIP
   port: 80
   targetPort: 8080
-
```

Comment:
hmm this is the detriment of using yq to process changes in the values.yaml... maybe it might make more sense to use `sed` for values.yaml operations to preserve newline formatting?

This is an open issue in yq noted in https://github.com/mikefarah/yq/issues/465.